elixir/wordcount.ex

```diff
@@ -0,0 +1,29 @@
+IO.stream(:stdio, :line)
+# Create a dictionary in which the keys are the words and the values are
+# the number of appearances in the input text.
+|> Enum.reduce(%{}, fn (line, acc) ->
```

Comment:
If I remember correctly, you are piping a Stream into an `Enum`-function, which does also forces the stream to completely evaluate before `Enum.reduce/3` can do something with it. But I have to admit, I can't come up with something better at the moment.